src/assets/css/potrait.css

```diff
@@ -0,0 +1,17 @@
+@media (max-width: 600px) {
```

Comment:
I know this will work but it's targeted for screen width less than 600px. but if we're distinguishing between portrait and landscape views, you could do `@media only screen and (orientation:portrait)` also

Reply:
I wouldn't think we'd want to limit to a specific pixel width, but rather design responsively, with a particular resolution in mind

Reply:
Pardon my ignorance :) but doesn't resolution=pixel width?  I believe this line is saying "apply these styles if the screen resolution is 600px or less wide?  Or likely I'm misunderstanding @mseaton 's comment.

Reply:
I do feel that this seems like the way to do it... to distinguish by pixel-width, not by whether the device is in landscape or portrait mode.

My thought is that this configuration would live in the existing "patientHeader.css"  in react-components, and we would/could/should (?) add other "@media" tags for other screen sizes?

@rhenshaw56 @jomadoye  do you have a sense of what best practice is here? is this responsive? :)

Reply:
I see, sorry I had misinterpreted it.  Seems right to me too